test/ClangImporter/no-nsobject-protocol.swift

```diff
@@ -0,0 +1,11 @@
+/// Don't crash when there's no NSObject protocol.
+/// rdar://problem/34597302
+
+// RUN: %target-swift-frontend -typecheck %s -import-objc-header %S/Inputs/no-nsobject-protocol.h
+
+// REQUIRES: objc_interop
```

Comment:
I think since there are no external dependencies you can use `-enable-objc-interop` instead.